main/ertp/guide/create-non-fungible-tokens.md

```diff
@@ -0,0 +1,79 @@
+# Creating non-fungible tokens with ERTP
+
+## Definition of fungible/non-fungible
+
+Assets are fungible if each asset is just as valuable as other assets of the same kind. 
+For instance, dollar bills are fungible because it doesn't matter which bill you use. 
+Each dollar bill is interchangeable with any other. Paintings, on the other hand, are not fungible. 
+Paintings have drastically different values depending on the content and painter. 
+A deal involving a specific painting would not be satisfied by another painting.
+
+
+## Creating a non-fungible asset with ERTP
+
+### Moldeing and creating the asset
```

Comment:
Let's change this to "Creating the asset." I think English-speaking readers will be unfamiliar with Moldeing as a word.

Reply:
oh... I think every reader from every language will have difficulty :-p
I meant "modeling" in the broad sense of https://en.wikipedia.org/wiki/Conceptual_model_(computer_science)

Thanks for catching the typo!